app/views/contribute/index.erb

```diff
@@ -157,7 +157,7 @@
           </p>
 
           <p>
-            Check out the <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#porting-an-exercise-to-another-language-track">detailed guide</a> if you want to give it a try!
+            Check out the <a href="https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#porting-an-exercise-to-another-language-track">detailed guide</a> if you want to give it a try!
```

Comment:
This got rewritten here: https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/porting-an-exercise.md